packages/palette-docs/content/docs/elements/Box.mdx

```diff
@@ -5,7 +5,7 @@ source: elements/Box
 
 <Playground>
   <Box>
-    <Text variant="md">
+    <Text variant="sm-display">
```

Comment:
What does "display" mean in this case?

Reply:
https://en.wikipedia.org/wiki/Display_typeface — generally: the version of this size that is suitable for headlines (not paragraphs). They just have tighter line heights.